### PR TITLE
Fix realms packages to be singular screen, map some other bits of it

### PR DESCRIPTION
**Pull request by i509VCB**

### CODE REVIEW

**Comment by liach** [comment]
honestly we should move these into net/minecraft/client/gui/screen/realms instead, given now these extend vanilla screen directly than using some sort of handle.

**Comment by i509VCB** [comment]
Yeah I could see the reasoning for truncating it to main screen package.

File: mappings/com/mojang/realmsclient/util.mapping [hunk 1]
```diff
@@ -1,4 +1,4 @@
-CLASS net/minecraft/class_4429 com/mojang/realmsclient/gui/screens/UploadResult
+CLASS net/minecraft/class_4429 com/mojang/realmsclient/util
```
Comment (YanisBft) [inline mappings/com/mojang/realmsclient/util.mapping:1]: Bad rename here

File: mappings/com/mojang/realmsclient/gui/screen/RealmsBackupScreen.mapping [hunk 2]
```diff
@@ -13,6 +13,8 @@ CLASS net/minecraft/class_4381 com/mojang/realmsclient/gui/screens/RealmsBackupS
 	FIELD field_19752 noBackups Ljava/lang/Boolean;
 	FIELD field_19753 serverData Lnet/minecraft/class_4877;
 	FIELD field_19754 titleLabel Lnet/minecraft/class_4903;
+	FIELD field_22686 PLUS_ICON_TEXTURE Lnet/minecraft/class_2960;
+	FIELD field_22687 RESTORE_ICON_TEXTURE Lnet/minecraft/class_2960;
```
Comment (YanisBft) [inline mappings/com/mojang/realmsclient/gui/screen/RealmsBackupScreen.mapping:17]:
```suggestion
	FIELD field_22686 PLUS_ICON Lnet/minecraft/class_2960;
	FIELD field_22687 RESTORE_ICON Lnet/minecraft/class_2960;
```

File: mappings/com/mojang/realmsclient/util.mapping (hunk 1 above)
Comment (liach) [reply mappings/com/mojang/realmsclient/util.mapping:1]: Ah, explains why java 11/14 build glitches out

**Comment by liach** [comment]
marking wip as there is broken renames.

**Comment by i509VCB** [comment]
I need to do this much more incrementally. This PR is HUGE. I'll open up a bunch of smaller PRs as time goes on

**Pull request closed without merging**